### ISSUE TITLE
Problems with emacs 26.0.50.2

### ISSUE THREAD

**Issue by LauBrust** [ergoemacs/ergoemacs-mode#435]
When I run
`(require 'ergoemacs-mode)`
After normal loads, I get the message:

```
Warning (ergoemacs): Could not find global map information
Loading /home/brust/.emacs.d/recentf...
Cleaning up the recentf list...
Cleaning up the recentf list...done (0 removed)
Ergoemacs-mode turned ON (us:standard).
Started ‘ergoemacs-mode’. Total startup time 4.018547 (Load: 2.863835, Initialize:1.154712)
Trailing garbage following expression:  (kill-emacs)
uncompressing ergoemacs-mode-26-0-50-2-i686-pc-linux-gnu.gz...
uncompressing ergoemacs-mode-26-0-50-2-i686-pc-linux-gnu.gz...done
compressing ergoemacs-mode-26-0-50-2-i686-pc-linux-gnu.gz...
compressing ergoemacs-mode-26-0-50-2-i686-pc-linux-gnu.gz...done
```

again and again at infinitum...
I've tried several things but unfortunately nothing seems to work.
A)
    1) Deleting ergoemacs-mode 
    2) Restarting Emacs
    3) Reinstalling ergoemacs-mode (from melpa: ergoemacs-mode-20161012.2127)
    4) Restarting emacs

My ergoemacs config is:

```
(setq ergoemacs-theme "standard")
(setq ergoemacs-keyboard-layout "es")
(require 'ergoemacs-mode)
(ergoemacs-mode 1)
```

and I am under GNU Emacs 26.0.50.2

B) I have the same problem running `emacs -Q`
    and eval the `*scratch*` buffer with

```
(setq debug-on-error t) ; Activate debugging
(add-to-list 'load-path "/mypath/to/ergoemacs/")
(require 'ergoemacs-mode)
(setq ergoemacs-debug t)
(setq ergoemacs-keyboard-layout "us") ; Layout you use.
(setq ergoemacs-theme nil) ; For standard theme.
(ergoemacs-mode 1)
```

This seems related to another issues, but I think that it is not equal, sorry if I am wrong.
Thank you very much.

**Comment by mattfidler** [comment]
To reproduce I need to find 32 bit binaries on windows.  Do you know where
they are at?

**Comment by Brusten** [comment]
Nop, I installed emacs from the repository `ppa:ubuntu-elisp/ppa`
I am sorry.

**Comment by mattfidler** [comment]
Emacs 25 is the stable; Why are you using emacs 26?

**Comment by mattfidler** [comment]
I cannot locate the binaries on google either.

**Comment by Brusten** [comment]
Well, I use emacs 26 because it is the only way that I know to solve a conflict that I have:
I want to use eww and maxima (with maxima-emacs), 
but the ubuntu current repository has emacs24 without eww and 
when I compile emacs25 and I install maxima-emacs, emacs24 is also installed as a dependency!
So I use this repository (`ppa:ubuntu-elisp/ppa`) and I can use both eww and maxima, but I have to use the unstable version of emacs.
Do you (or someone else) know if there a repository with the latest stable emacs version?

I will try again compiling the source code of emacs.
Thank you and sorry.

**Comment by mattfidler** [comment]
No.  I do not.  

However this page may be helpful:
http://ubuntuhandbook.org/index.php/2016/09/install-gnu-emacs-25-1-in-ubuntu-16-04/

**Comment by mattfidler** [comment]
It seems that the emacs24 dependencies could be due to the build requirements being the same...?

**Comment by Brusten** [comment]
Thank you very much, `checkinstall` is the part I was missing! Now, I don't have the conflict.

Yes, the build requirements are the same for emacs24 and emacs25.1.

Thanks again.

**Comment by mattfidler** [comment]
I think this should also work with emacs 26.  I am closing this for now.

**Comment by ceridwen** [comment]
For what it's worth, I ran into this error when Homebrew updated Emacs from 25.3 to 26.1 without my noticing.  Reinstalling the package didn't seem to help, whenever I tried to start ergoemacs mode in 26.1, it would halt at `Trailing garbage following expression:  (kill-emacs)` and I'd have to kill the process.  Downgrading to 25.3 fixed the issue.

**Comment by fpvmorais** [comment]
Can confirm the same issue happens on Windows 64 bits - Emacs 26

**Comment by mattfidler** [comment]
I'm running Widows 64 bits emacs 26 and have no issue.

**Comment by mattfidler** [comment]
How did you install. I think the elpa mirror isn't working.  I need to reset my account and remember how to upload ergomacs-mode there.

**Comment by fpvmorais** [comment]
I installed via melpa. I'll try use the repo version. Thanks for the quick reply and for your amazing work :)

**Comment by mattfidler** [comment]
Thanks @juster.  I think I need to revisit this;  Did you install from elpa.

**Comment by juster** [comment]
The 5.16.10.12 package from either melpa-stable or gnu have the same bug. Just to reiterate, the child process hangs when the global keymap file is dynamically generated the first time ergoemacs is turned on:

```
Warning (ergoemacs): Could not find global map information
Loading c:/Users/jrcd/AppData/Roaming/.emacs.d/recentf...
Cleaning up the recentf list...
Cleaning up the recentf list...done (0 removed)
Ergoemacs-mode turned ON (us:standard).
Started `ergoemacs-mode'. Total startup time 2.112636 (Load: 1.442886, Initialize:0.669750)
Trailing garbage following expression:  (kill-emacs)
```

After the last line, Emacs hangs indefinitely and the startup time keeps increasing. I resolved this by cloning the git repo and using it directly. The ergoemacs 20181127.2130 package from melpa avoids the hang as well, presumably because it is a direct copy of the latest git repo source.

In other words I believe a new release is needed for the gnu & melpa-stable package archives. This bug with emacs 26/win32 is fixed it has just not been released yet.

**Comment by paragjoshi** [comment]
I confirm that I had the same problem after installing emacs using apt-get on ubuntu 20.04 and installing ergoemacs with ELPA /  Melpa.
I could make it work only by doing manual install. 
Thank you for contributions. This is a great package.
